srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
2160
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh03-01n32: task 0: Exited with exit code 1
srun: Terminating job step 9429246.10
2160
2160
srun: error: sh03-01n31: task 0: Exited with exit code 1
srun: Terminating job step 9429246.3
srun: error: sh03-01n32: task 0: Exited with exit code 1
srun: Terminating job step 9429246.11
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
2160
srun: error: sh03-01n32: task 0: Exited with exit code 1
srun: Terminating job step 9429246.9
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh03-01n32: task 0: Exited with exit code 1
srun: Terminating job step 9429246.12
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh03-01n31: task 0: Exited with exit code 1
srun: Terminating job step 9429246.6
srun: error: sh03-01n32: task 0: Exited with exit code 1
2160
srun: Terminating job step 9429246.13
srun: error: sh03-01n32: task 0: Exited with exit code 1
srun: Terminating job step 9429246.8
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh03-01n31: task 0: Exited with exit code 1
2160
srun: Terminating job step 9429246.4
srun: error: sh03-01n31: task 0: Exited with exit code 1
srun: error: sh03-01n31: task 0: Exited with exit code 1
srun: Terminating job step 9429246.7
srun: Terminating job step 9429246.5
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh03-01n33: task 0: Exited with exit code 1
srun: Terminating job step 9429246.14
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh03-01n33: task 0: Exited with exit code 1
srun: Terminating job step 9429246.15
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh03-01n33: task 0: Exited with exit code 1
srun: Terminating job step 9429246.16
srun: error: sh03-01n33: task 0: Exited with exit code 1
srun: Terminating job step 9429246.19
2160
srun: error: sh03-01n33: task 0: Exited with exit code 1
srun: error: sh03-01n33: task 0: Exited with exit code 1
srun: Terminating job step 9429246.17
srun: Terminating job step 9429246.18
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh03-01n29: task 0: Exited with exit code 1
srun: Terminating job step 9429246.0
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh03-01n29: task 0: Exited with exit code 1
srun: error: sh03-01n29: task 0: Exited with exit code 1
srun: Terminating job step 9429246.2
srun: Terminating job step 9429246.1
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh03-01n34: task 0: Exited with exit code 1
srun: Terminating job step 9429246.20
srun: error: sh03-01n34: task 0: Exited with exit code 1
srun: Terminating job step 9429246.22
srun: error: sh03-01n34: task 0: Exited with exit code 1
srun: Terminating job step 9429246.21
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh03-01n35: task 0: Exited with exit code 1
srun: error: sh03-01n35: task 0: Exited with exit code 1
srun: Terminating job step 9429246.23
srun: Terminating job step 9429246.24
2160
2160
2160
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh03-01n38: task 0: Exited with exit code 1
srun: Terminating job step 9429246.28
srun: error: sh03-01n38: task 0: Exited with exit code 1
srun: error: sh03-01n38: task 0: Exited with exit code 1
srun: error: sh03-01n38: task 0: Exited with exit code 1
srun: error: sh03-01n38: task 0: Exited with exit code 1
srun: Terminating job step 9429246.25
srun: Terminating job step 9429246.26
srun: Terminating job step 9429246.29
srun: Terminating job step 9429246.27
